src/header/Header.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as React from "react";
+import {ReactNode} from "react";
```

Comment:
Why need 2 import statements for react ? you have already imported everything from 'react'.  You can access ReactNode as React.ReactNode.

Reply:
Done - removed extra import